--- original.setup.py	2025-05-17 23:12:13.744606624 +0200
+++ setup.py	2025-05-17 23:17:38.800001189 +0200
@@ -16,7 +16,7 @@
     'scipy',
     'shapely',
     'alphashape',
-    'py3dtiles @ git+https://gitlab.com/py3dtiles/py3dtiles@main',
+    'py3dtiles @ git+https://gitlab.com/py3dtiles/py3dtiles@aa04d83e',
     'py3dtiles_temporal_extension @ git+https://gitlab.com/VCityTeam/py3dtiles_temporal_extension',
     'earclip @ git+https://github.com/lionfish0/earclip',
     'Pillow',
